.gitignore

```diff
@@ -114,3 +114,9 @@ cmd/weaviate-server/__debug_bin*
 venv
 **/__pycache__
 *.pyc
+
+# weaviate binaries
+cmd/weaviate/weaviate
+/weaviate
+cmd/weaviate-server/weviate-server
+weaviate-server
```

Comment:
are all of the ignores valid? actually I don't see `/weaviate` folder in my repo, or `weaviate-server` files in root

Reply:
Ah. I think it got from the `main` branch. (I was switching branches). Fixed